tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/defaults/main.yml

```diff
@@ -111,3 +111,7 @@ vault_key_file: '{{ local_temp_dir }}/privatekey.pem'
 
 vault_proxy_server: 'http://127.0.0.1:8001'
 vault_proxy_external: False
+
+# if any connections are taking longer than this to complete there's probably something really wrong
+# with the integration tests, so it'd be better to fail faster than the 30s default
+ansible_hashi_vault_timeout: 5
```

Comment:
Very nice - I wonder if calling that in the description would be beneficial. Although you would reference a value from another project that is a moving target.

Reply:
I had it in there, when I had set the plugin's default for the value to 30s, but I thought I'd rather it "fall through" instead of having its own default, I just didn't update that comment.